eventsourced/src/lib.rs

```diff
@@ -94,11 +94,36 @@ where
     /// The command handler, taking this command, and references to the ID and the state of
     /// the event sourced entity, either rejecting this command via [Self::Error] or returning an
     /// event.
```

Comment:
I guess we should update the rustdoc here

Reply:
Will do.

Reply:
Done